packages/typegpu/src/data/align.ts

```diff
@@ -64,7 +64,7 @@ class AbstractTgpuAlignedImpl<
   public readonly isCustomAligned = true;
 
   constructor(
-    private data: AnyTgpuData | AnyTgpuLooseData,
+    protected data: AnyTgpuData | AnyTgpuLooseData,
```

Comment:
```suggestion
    protected data: TData,
```
that way we can eliminate checking if data is loose inside TgpuAligned

Reply:
Great idea